puffin/src/lib.rs

```diff
@@ -852,29 +1238,26 @@ macro_rules! profile_function {
 /// An optional second argument can be a string (e.g. a mesh name) to help diagnose what was slow.
 /// Example: `profile_scope!("load_mesh", mesh_name);`
 ///
-/// Overhead: around 140 ns on 2020 Intel MacBook Pro.
+/// Overhead: around 54 ns on Macbook Pro with Apple M1 Max.
 #[macro_export]
 macro_rules! profile_scope {
     ($id:expr) => {
         $crate::profile_scope!($id, "");
     };
     ($id:expr, $data:expr) => {
```

Comment:
do we want to intern `data` as well?

Reply:
intern?

Reply:
hmm interesting, logically it can be completely dynamic data, but typically it is semi static. so doing string interning on it also could be benefit for most cases.  though this is used a lot less so think skip it for this PR and change

Reply:
I thought `data` was the literal used for the scope. But I guess that is `id`. What happens to `id` now? do we ignore it?

Reply:
id is scope_name now and is only send once. I think we can follow up on the move to more static strings. This is something Niklas had a PR for before

Reply:
But what about the macro argument `id`? It looks to me like that is discarded? Isn't that supposed to be the name of the scope that is to be shown in the gui?

Reply:
Good point! I am not sending/sharing it atm, need to fix that.